### ISSUE TITLE
Removing sign up option from drop-in component

### ISSUE THREAD

**Issue by ADringer** [aws-amplify/amplify-js#2946]
** Which Category is your question related to? **
amplify-authenticator drop-in

** What AWS Services are you utilizing? **
Cognito

** Provide additional details e.g. code snippets **
In Cognito we've configured our user pool policy to only allow administrators to create users. We are now implementing this within our application and are using the amplify-authenticator component from the aws-amplify-angular package. 

However, this is displaying a link to sign up. Is there a way to remove this as sign up is blocked by the policy? I couldn't see anything in the `Amplify.configure` object or the binding properties of the component.

Thanks

**Comment by gerrytsui** [comment]
The only solution I found, other than modifying the library source code, is to add this in the styles.css ( that is referenced in my angular.json)

.amplify-form-signup {
    display:none;
}

**Comment by powerful23** [comment]
Duplicated to #1191

**Comment by jordanranz** [comment]
Closing this as it is a duplicate of #1191. There is a PR to fix this.
